boards/shields/nrf7002ek_nrf7002_eb/nrf7002ek_nrf7002_eb.overlay

```diff
@@ -0,0 +1,27 @@
+/* Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <freq.h>
+#include "nrf7002ek_nrf7002_eb_coex.overlay"
+
+&edge_connector_spi {
+	status = "okay";
+	cs-gpios = <&gpio0 11 GPIO_ACTIVE_LOW>;
```

Comment:
This should be defined in the board dts, where the `edge_connector_spi` label is also introduced.